tools/karpenter-convert/README.md

```diff
@@ -9,7 +9,21 @@ It converts `v1alpha5/Provisioner` to `v1beta1/NodePool` and `v1alpha1/AWSNodeTe
 go install github.com/aws/karpenter/tools/karpenter-convert/cmd/karpenter-convert@latest
 ```
 
-## Usage:
+## Usage
+```console
+Usage:
+  karpenter-convert [flags]
+
+Flags:
+  -f, --filename strings               Filename, directory, or URL to files to need to get converted.
+  -h, --help                           help for karpenter-convert
+  -I, --ignore-defaults                Ignore defining default requirements when migrating Provisioners to NodePool.
+  -k, --kustomize string               Process the kustomization directory. This flag can't be used together with -f or -R.
```

Comment:
We don't have any tests for this behavior so should we either: 
1. Validate that this works through testing or
2. Drop this behavior from our tool